src/Assets/Hathora/Core/Scripts/Runtime/Client/HathoraClientSession.cs

```diff
@@ -75,7 +75,7 @@ public void InitNetSession(string playerAuthToken)
             #region (!) TEMPORARY WORKAROUND FOR CLIENT AUTH TOKEN --Dylan
             Debug.Log($"[HathoraClientSession.{nameof(InitNetSession)}] <color=orange>(!) SDK WORKAROUND: " +
                 "Adding client Authorization Token header to temporary `SDKConfig.ClientAuthToken` --Dylan</color>");
-            SDKConfig.ClientAuthToken = playerAuthToken;
```

Comment:
This should be able to be removed @dylanh724 , now that we have `PlayerAuth` securities